elide-core/src/main/java/com/yahoo/elide/resources/JsonApiEndpoint.java

```diff
@@ -112,13 +113,29 @@ public Response patch(
      */
     @DELETE
     @Path("{path:.*}")
+    @Consumes("application/vnd.api+json")
     public Response delete(
         @PathParam("path") String path,
         @Context SecurityContext securityContext,
         String jsonApiDocument) {
         return build(elide.delete(path, jsonApiDocument, getUser.apply(securityContext)));
     }
 
+    /**
+     * Delete resource handler (expects no body and no content-type header).
+     *
+     * @param path request path
+     * @param securityContext security context
+     * @return response
+     */
+    @DELETE
+    @Path("{path:.*}")
+    public Response deleteWithoutBody(
```

Comment:
Perhaps this should be `delete` and the other should be renamed to `deleteRelationships`? If that makes sense this would be better suited to going into the `elide3.0` branch.

Thoughts @DeathByTape?